docs/third-party-developers/extensibility/data-store/validation.md

```diff
@@ -103,9 +103,30 @@ const store = dispatch( 'wc/store/validation' );
 store.clearValidationError( 'billing-first-name' );
 ```
 
-### clearAllValidationErrors
+### clearValidationErrors
 
-Clears all validation errors.
+Clears multiple validation errors at once. If no error IDs are passed, all validation errors will be cleared.
+
+#### _Parameters_
+
+- _errors_ `string[] | undefined` - The error IDs to clear validation errors for. This can be undefined, and if it
+  is, all validation errors will be cleared.
```

Comment:
Something funky going on with the spacing here. Can you check prettier runs on this file?

Reply:
Nice one thanks Alex, I've updated that now.